policyengine_us/parameters/gov/household/household_benefits.yaml

```diff
@@ -18,6 +18,7 @@ values:
     - basic_income
     - spm_unit_capped_housing_subsidy
     - household_state_benefits
+    - commodity_supplemental_food_program
```

Comment:
could you add it to a new 2024 list instead:
```
  2024-01-01:
    - social_security
    - ssi
    - snap
    - wic
    - free_school_meals
    - reduced_price_school_meals
    - spm_unit_broadband_subsidy
    - tanf
    - high_efficiency_electric_home_rebate
    - residential_efficiency_electrification_rebate
    - unemployment_compensation
    - head_start
    - early_head_start
    # Contributed.
    - basic_income
    - spm_unit_capped_housing_subsidy
    - household_state_benefits
    - commodity_supplemental_food_program
```

Reply:
Done. The Commodity Supplemental Food Program has existed before this year, I just did not check that the rules have been the same.

Reply:
Filed an [issue](https://github.com/PolicyEngine/policyengine-us/issues/5035)